data_sources/kafka_data_source.py

```diff
@@ -0,0 +1,61 @@
+from data_sources.data_source import DataSource
+from confluent_kafka import Consumer, KafkaError
+import time
+import json
+
+
+class KafkaDataSource(DataSource):
+    def __init__(self, kafka_endpoint, kafka_topic, s3_bucket, read_timeout_secs, batch_size = 1024):
+        super().__init__(s3_bucket)
```

Comment:
It's better to use `s3_bucket=s3_bucket` in case the parent function change

Reply:
I think its better always to call the super function

Reply:
Of course, what I meant is:
`super().__init__(s3_bucket=s3_bucket)`

Reply:
Done